### PR TITLE
updated raindrop (latest)

### PR DESCRIPTION
**Pull request by enavarrocu**
#### Editing an existing cask
- [x] Commit message includes cask’s name (and new version, if applicable).
- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` left no offenses.

### CODE REVIEW

File: Casks/raindrop.rb [hunk 1]
```diff
@@ -3,9 +3,9 @@
   sha256 :no_check
 
   url 'https://raindrop.io/releases/mac/Raindrop.dmg'
-  name 'Raindrop'
+  name 'Raindropio'
```
Comment (reitermarkus) [inline Casks/raindrop.rb]: This should be `Raindrop.io`.

Comment (enavarrocu) [inline Casks/raindrop.rb]: For some reason, it didn't change it because I did the change.

**Comment by reitermarkus** [comment]
Your pull request was merged directly in 9ba1789522e505c7f3a0f51fce73a791bd3b558d to keep commit history cleaner. It needed some changes which were made in 9ba1789522e505c7f3a0f51fce73a791bd3b558d. Thank you for your contribution!

**Comment by reitermarkus** [comment]
Also, the token had to be changed: https://github.com/caskroom/homebrew-cask/commit/7f86044fc315a8e32f1b171a34f8dbf6b01f5acb

**Pull request closed without merging**